packages/account/src/providers/transaction-request/utils.test.ts

```diff
@@ -42,3 +45,88 @@ describe('isTransactionTypeBlob', () => {
     expect(isTransactionTypeCreate(request)).toBe(false);
   });
 });
+
+describe('transactionRequestify', () => {
```

Comment:
May be worth cloning with some inputs on the `baseRequest`:

```ts
const baseRequest = new ScriptTransactionRequest();
newRequest.inputs.push(baseInput);
expect(baseRequest.inputs).toHaveLength(1);

const newRequest = ScriptTransactionRequest.from(baseRequest);
expect(newRequest).to.not.equal(baseRequest);
expect(newRequest.inputs).toHaveLength(1);

newRequest.inputs.push(newInput);
expect(newRequest.inputs).to.not.equal(baseRequest.inputs);
```

Reply:
I don't think we need extensive tests on the [clone](https://ramdajs.com/docs/#clone) method since it's from an external library, I only added these tests so that should someone remove the _clone_ functionality, we would catch it.